parseutil/parseutil.go

```diff
@@ -106,19 +106,28 @@ func ParseDurationSecond(in interface{}) (time.Duration, error) {
 			return dur, nil
 		}
 		var err error
-		// Look for a suffix otherwise its a plain second value
-		if strings.HasSuffix(inp, "s") || strings.HasSuffix(inp, "m") || strings.HasSuffix(inp, "h") || strings.HasSuffix(inp, "ms") {
+
+		suffix := inp[len(inp)-1:]
```

Comment:
I think this will be OK as is. All [UTF-8 encoded Unicode](https://en.wikipedia.org/wiki/UTF-8#Encoding) are made up of bytes >= 0x80, so if part of a multi-byte character is sliced off the remaining string will have a high byte that will fail to parse.